cmd/wasm-client/go.mod

```diff
@@ -2,18 +2,20 @@ module github.com/lightninglabs/lightning-node-connect/cmd/wasm-client
 
 require (
 	github.com/btcsuite/btcd/btcec/v2 v2.3.4
-	github.com/btcsuite/btclog v0.0.0-20241003133417-09c4e92e319c
+	github.com/btcsuite/btclog/v2 v2.0.1-0.20250110154127-3ae4bf1cb318
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/faraday v0.2.13-alpha
-	github.com/lightninglabs/lightning-node-connect v0.3.3-alpha.0.20250303090341-f823ad04b9de
-	github.com/lightninglabs/lightning-terminal v0.14.0-alpha
-	github.com/lightninglabs/loop v0.29.0-beta
-	github.com/lightninglabs/pool v0.6.5-beta.0.20241015105339-044cb451b5df
-	github.com/lightninglabs/taproot-assets v0.5.0
-	github.com/lightningnetwork/lnd v0.18.4-beta
+	github.com/lightninglabs/faraday v0.2.14-alpha.0.20250305102803-f668619b351e
+	github.com/lightninglabs/lightning-node-connect/gbn v1.0.0
+	github.com/lightninglabs/lightning-node-connect/mailbox v1.0.0
+	github.com/lightninglabs/lightning-terminal/litrpc v1.0.2-0.20250506112654-08095d30dd1a
+	github.com/lightninglabs/lightning-terminal/perms v0.0.0-20250506112654-08095d30dd1a
+	github.com/lightninglabs/loop/looprpc v1.0.7
+	github.com/lightninglabs/pool v0.6.5-beta.0.20250305125211-4e860ec4e77f
```

Comment:
Should this be `poolrpc` :)?

Reply:
Good question! For consistency I will move `perms` in pool into `poolrpc`. And then this wasm-client dep can just be `poolrpc`.

Reply:
Created PR: https://github.com/lightninglabs/pool/pull/503

Reply:
Awesome, thanks :rocket:!